gnucash/import-export/ofx/gnc-ofx-import.cpp

```diff
@@ -1332,20 +1338,21 @@ runMatcher (ofx_info* info, char * selected_filename, gboolean go_to_next_file)
                 g_free (amtstr);
                 g_free (datestr);
             }
-            trans_list_remain = g_list_prepend (trans_list_remain, trans);
-            g_free (date_amount_key);
+            trans_list_remain.emplace_back (trans);
         }
         else
         {
-            g_hash_table_insert (trans_hash, date_amount_key, account);
+            trans_map.emplace (date_amount_key, account);
```

Comment:
`insert`, nothing to construct.

Reply:
This one does need `emplace`; the insert form will read `trans_map.insert ({date_amount_key, account});`.